File: app/src/main/java/com/inguana/weatherapp/utils/ErrorMessageSnackbar.java
package com.inguana.weatherapp.utils;

import android.view.View;

import com.google.android.material.snackbar.Snackbar;

public class ErrorMessageSnackbar {

    public static void showErrorMessage(View rootLayout, String errorMessage) {
        Snackbar snackbar = Snackbar
                .make(rootLayout, errorMessage, Snackbar.LENGTH_LONG);
        snackbar.show();
    }
}

File: app/src/main/java/com/inguana/weatherapp/utils/ErrorMessageSnackbar.kt
package com.inguana.weatherapp.utils

import android.view.View
import com.google.android.material.snackbar.Snackbar

object ErrorMessageSnackbar {
    fun showErrorMessage(rootLayout: View?, errorMessage: String?) {
        val snackbar = Snackbar
            .make(rootLayout!!, errorMessage!!, Snackbar.LENGTH_LONG)
        snackbar.show()
    }
}